docker-compose.yaml

```diff
@@ -164,58 +169,25 @@ services:
       - "traefik.http.routers.play.priority=1"
       - "traefik.http.routers.play.service=play"
       - "traefik.http.services.play.loadbalancer.server.port=3000"
-
-  chat:
```

Comment:
Don't forgot to update README ('/etc/hosts' ...)

Reply:
Done